### ISSUE TITLE
Execution of MNIST freezes

### ISSUE THREAD

**Issue by itsergiu** [keras-team/autokeras#619]
Environment, VirtualBox:
https://github.com/Virtual-Machines/Anaconda-VirtualBox/issues/2

`tensorflow.__version__`
'1.12.0'
`keras.__version__`
'2.2.4'

Execution of autokeras MNIST code freezes on Training model 1.
I have executed autokeras MNIST code several times in Spyder and Jupyter notebook and same freeze occurs.
How can I solve it?

+----------------------------------------------+
|               Training model 1               |
+----------------------------------------------+
Epoch-1, Current Metric - 0:   2%|▌                            | 10/465 [00:05<04:30,  1.68 batch/s]

**Comment by DavidGarciaPrados** [comment]
Hi, @itsergiu 
Is this example?: https://raw.githubusercontent.com/keras-team/autokeras/master/examples/a_simple_example/mnist.py

I have tested this code and requires a lot of resources. You can try to increase the RAM assigned to machine, now has 2048 MB ![VBoxMemory](https://user-images.githubusercontent.com/6674996/56499155-70030a00-6505-11e9-806a-467a937df9fa.png) 

However, in this situation, **is preferable real hardware than my ".ova"** because you can use all the CPU cores, memory and the GPU:

> Enable Multi-GPU Training
Auto-Keras support multiple GPU training in the default setting. There's no additional step needed to enable multiple GPU training. However, if multiple-GPU training is not a desirable behavior. You can disable it via environmental variable. CUDA_VISIBLE_DEVICES. For example, in your bash: export CUDA_VISIBLE_DEVICES=0. Keep in mind that when using multiple-GPU, make sure batch size is big enough that multiple-gpu context switch overhead won't effect the performance too much. Otherwise multiple-gpu training may be slower than single-GPU training.

Autokeras can be installed with pip. Useful links:

Get Python 3.6: https://www.python.org/downloads/

Download pip tutorial: https://www.liquidweb.com/kb/install-pip-windows/ 

pip command for Autokeras (https://autokeras.com/start/): **pip install autokeras**

Installing Jupyter (https://jupyter.org/install) and Spyder (https://docs.spyder-ide.org/installation.html) is easy without Anaconda too.

Hope this helps.

**Comment by itsergiu** [comment]
I already assigned to machine 4096 MB and 4 CPU.
I will test also what you recommended.
I tested code from https://autokeras.com/start/ 
```
from keras.datasets import mnist
from autokeras.image.image_supervised import ImageClassifier

if __name__ == '__main__':
    (x_train, y_train), (x_test, y_test) = mnist.load_data()
    x_train = x_train.reshape(x_train.shape + (1,))
    x_test = x_test.reshape(x_test.shape + (1,))`

    clf = ImageClassifier(verbose=True)
    clf.fit(x_train, y_train, time_limit=12 * 60 * 60)
    clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
    y = clf.evaluate(x_test, y_test)
    print(y)
```

**Comment by taylor4712** [comment]
> Hi, @itsergiu
> Is this example?: https://raw.githubusercontent.com/keras-team/autokeras/master/examples/a_simple_example/mnist.py
> 
> I have tested this code and requires a lot of resources. You can try to increase the RAM assigned to machine, now has 2048 MB ![VBoxMemory](https://user-images.githubusercontent.com/6674996/56499155-70030a00-6505-11e9-806a-467a937df9fa.png)
> 
> However, in this situation, **is preferable real hardware than my ".ova"** because you can use all the CPU cores, memory and the GPU:
> 
> > Enable Multi-GPU Training
> > Auto-Keras support multiple GPU training in the default setting. There's no additional step needed to enable multiple GPU training. However, if multiple-GPU training is not a desirable behavior. You can disable it via environmental variable. CUDA_VISIBLE_DEVICES. For example, in your bash: export CUDA_VISIBLE_DEVICES=0. Keep in mind that when using multiple-GPU, make sure batch size is big enough that multiple-gpu context switch overhead won't effect the performance too much. Otherwise multiple-gpu training may be slower than single-GPU training.
> 
> Autokeras can be installed with pip. Useful links:
> 
> Get Python 3.6: https://www.python.org/downloads/
> 
> Download pip tutorial: https://www.liquidweb.com/kb/install-pip-windows/
> 
> pip command for Autokeras (https://autokeras.com/start/): **pip install autokeras**
> 
> Installing Jupyter (https://jupyter.org/install) and Spyder (https://docs.spyder-ide.org/installation.html) is easy without Anaconda too.
> 
> Hope this helps.

Where can I find the autokeras settings? How and where did you increase the RAM?
